lib/config.coffee

```diff
@@ -3,19 +3,28 @@ module.exports =
     type: 'string'
     default: 'python'
     order: 1
+    description: 'A path to the python executable to launch the ycmd server process with.'
   ycmdPath:
     type: 'string'
     default: '/usr/lib/youcompleteme/third_party/ycmd'
     order: 2
+    description: 'The directory containing the `ycmd/default_settings.json` file.'
   enabledFiletypes:
     type: 'string'
     default: 'c, cpp, objc, objcpp'
     order: 3
+    description: 'A comma-separated list of file-extensions within we should provide suggestions.'
```

Comment:
they are not file extensions. they are the language components of the editor's root scope descriptor. atom doesn't have much documentation on this yet. https://github.com/atom/atom/issues/8430
here is a table for reference(see entries like `.source.js`): https://github.com/atom/autocomplete-plus/wiki/Autocomplete-Providers